### ISSUE TITLE
Does Cilium support AWS EKS Fargate?

### ISSUE THREAD

**Issue by s0rl0v** [cilium/cilium#15315]
Hello team,

Does Cilium support AWS EKS Fargate? It's not listed anywhere in docs, so I hope to find the answer here.

**Comment by pchaigno** [comment]
No, I think we're blocked on https://github.com/aws/containers-roadmap/issues/1027.

**Comment by pchaigno** [comment]
Closing in favor of https://github.com/aws/containers-roadmap/issues/1027.
